packages/babel-plugin-transform-flow-comments/test/fixtures/flow-comments/arrow-single-param-type/output.js

```diff
@@ -1 +1 @@
-const x = (foo /*: string*/) => {};
+const x = foo /*: string*/ => {};
```

Comment:
This is not a valid Flow comment. If the arrow function has a single parameter with a trailing block comment that starts with `:`, we should still print the parentheses.

Reply:
Sad that flow does not support that. In that case we can partially revert the arrow function commit.

Reply:
There doesn't seem to be a change here?

Reply:
Ah I forgot to update the plugin test fixtures.